docs/runtimes/job-function.md

```diff
@@ -49,6 +49,21 @@ To run the job:
 project.run_function("train")
 ```
 
+You can configure automatic retries for failed job runs by passing a `retry` dictionary
+when running a function. This enables MLRun to automatically handle transient errors such as runtime failures,
+OOM, or pod evictions.
+
+Example:
+```python
+project.run_function(
+    "train",
+    retry={
+        "count": 3,  # total retries allowed
+        "backoff": {"base_delay": 30},  # delay in seconds between retries
+    },
+)
+```
+
```

Comment:
You can view:
- UI: The retry status in the **Jobs and Workflows > Monitor Jobs** page under the Retries column. 
- UI: Pending retries in the **Jobs and Workflows > Monitor Jobs**
- UI The Logs per attempt in the Logs tab, by selecting an attempt from the drop-down list.
- Log collector: The Logs per attempt
The final notification (success/failure) is sent after the last attempt, including the number of attempted retries.